src/main/kotlin/no/nav/syfo/dialogmotestatusendring/DialogmoteStatusendringQueries.kt

```diff
@@ -0,0 +1,69 @@
+package no.nav.syfo.dialogmotestatusendring
+
+import no.nav.syfo.database.toList
+import no.nav.syfo.dialogmotestatusendring.domain.DialogmoteStatusendring
+import no.nav.syfo.dialogmotestatusendring.domain.PDialogmoteStatusendring
+import java.sql.*
+import java.time.Instant
+import java.time.OffsetDateTime
+import java.util.*
+
+const val queryCreateDialogmoteStatusendring =
+    """INSERT INTO dialogmote_statusendring (
+        id,
+        uuid,
+        mote_uuid,
+        arbeidstaker_ident,
```

Comment:
Hva tenker vi om rutiner for å håndtere identhendelser på nye tabeller? Denne tabellen inneholder jo også fnr, som gjør at den burde oppdateres ved nye identer. Burde det være den som utvider tabellen sin ansvar å passe på dette?

Det er litt tungvint at man må huske på det hver gang man lager en ny tabell 🤔

Reply:
Vi lager en egen PR på det, vi må legge inn oppdatering av møtestatus også.

Reply:
Vi legger inn en oppdatering på det i en egen pull request. Vi må også oppdatere tabellen motesvar, den har personident.